src/mudlet.cpp

```diff
@@ -508,6 +515,12 @@ void mudlet::initEdbee()
     // We only need the single Lua lexer, probably ever
     grammarManager->readGrammarFile(QLatin1Literal(":/edbee_defaults/Lua.tmLanguage"));
 
+    //Open and parse luaFunctionList
```

Comment:
Not a very good comment - the functions name is luaFunctionList to begin with - what is this list?

Reply:
I'll make it more insightful